### ISSUE TITLE
 Insecta Mundi template problem

### ISSUE THREAD

**Issue by truschel** [gsautter/goldengate-imagine#861]
Hi Guido

I am working for create a template from 2017 to 2019 of Insecta Mundi journal, and I checked all papers of this period. In this period the journal has two papers with the image next to the text and two papers with one more cover page (the pattern is two). So, could I made individual extractions for these four papers and continue with the template creation for the others?

**Comment by gsautter** [comment]
Exactly. Or you can even run these papers alongside the others, just have to keep in mind to do some manual correction afterwards.

On a more general note, I'm left to wonder why a journal would vary its cover pages an figure alignment ... or is this simply sloppiness?
